api/Properties/launchSettings.json

```diff
@@ -3,15 +3,14 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:9897",
-      "sslPort": 44350
+      "applicationUrl": "http://localhost:5000;http://host.docker.internal:5000"
     }
   },
   "profiles": {
     "api": {
       "commandName": "Project",
       "launchBrowser": true,
```

Comment:
Pls don't change the current profile to work with docker by default, create a new profile instead.